vignettes/submodel_documentation_overview.Rmd

```diff
@@ -0,0 +1,394 @@
+-`prob_strand_early` & `prob_strand_late`: Expert estimated stranding probabilites. 
+-`proportion_diverted` & `total_diverted`: Proportion and total flows diverted from DSMflow data package. More details at [proportion_diverted](https://reorienting-to-recovery.github.io/DSMflow/reference/proportion_diverted.html)
+-`weeks_flooded`: More details at weeks_flooded
+-`prop_high_predation`: More details at prop_high_predation
+-`contact_points`: More details at contact_points
+-`..surv_juv_rear_int`: Intercept, source: calibration (varies by tributary)
+-`.surv_juv_rear_contact_points`: Coefficient for contact_points variable, source: inherited from previous calibration
+-`..surv_juv_rear_contact_points`: Coefficient for contact_points variable, source: calibration
+-`.surv_juv_rear_prop_diversions`: Coefficient for prop_diversions variable, source: Newman and Brandes 2010
+-`..surv_juv_rear_prop_diversions`: Coefficient for prop_diversions variable, source: calibration
+-`.surv_juv_rear_total_diversions`: Coefficient for total_diversions variable, source: inherited from previous calibration
+-`..surv_juv_rear_total_diversions`: Coefficient for total_diversions variable, source: calibration
+-`.surv_juv_rear_avg_temp_thresh`: Coefficient for avg_temp_thresh variable, source: Runge et al 2008
+-`.surv_juv_rear_high_predation`: Coefficient for high_predation variable, source: Cavallo et al. 2012
+-`.surv_juv_rear_stranded`: Coefficient for stranded variable, source: USFWS 2006 and CDWR 2006
+-`.surv_juv_rear_medium`: Size related intercept for medium sized fish, source: [Connor et al. 2004](https://afspubs.onlinelibrary.wiley.com/doi/abs/10.1577/M02-161.1)
+-`.surv_juv_rear_large`: Size related intercept for large sized fish, source: [Connor et al. 2004](https://afspubs.onlinelibrary.wiley.com/doi/abs/10.1577/M02-161.1)
+-`.surv_juv_rear_floodplain`: Additional intercept for floodplain rearing benefit, source: Sommer et al. (2001)`
+
+
+#### Bypass
+Bypass survival is a function of the temperatures, predation, habitat type, and size of fish. This function also contains a calibrated intercept.  In the model, this information is contained in the following data objects: 
+
+* data: `average_temp`  
+* data: `max_temp_thresh`
+* data: `avg_temp_thresh`
+* data: `high_predation`
+* calibrated intercept: `..surv_juv_bypass_int`
+* coefficient: `.high_predation`
+* coefficient: `.medium`
+* coefficient: `.large`
+* coefficient: `.floodplain`
+* coefficient: `min_survival_rate`
+
+and it is input to the `fall_run_model` as part of the params list. 
+
+
+**Source:**
+* `average_temp` More details at stream_temperature
+* `max_temp_thresh` Variable representing the probability of exceeding the max temp threshold. Coefficient for high_predation variable, source: [Cavallo et al. 2012] (https://search.nal.usda.gov/discovery/search?query=lds35,contains,512123-01nal_inst,AND&tab=LibraryCatalog&search_scope=MyInstitution&vid=01NAL_INST:MAIN&mode=advanced&offset=0)
```

Comment:
this source formatting isn't working - delete space between brackets and parentheses

Reply:
fixed